docs/advanced-features/using-mdx.md

```diff
@@ -0,0 +1,166 @@
+---
+description: Learn how to use @next/mdx in your Next.js project.
+---
+
+# Using MDX with Next.js
+
+Next.js supports MDX through a number of different means, this page will outline some of the ways you can begin integrating MDX into your Next.js project.
+
+## What is MDX?
+
+MDX is a superset of markdown that lets you write JSX directly in your markdown files. It is a powerful way to add dynamic interactivity, and embed components within your content, helping you to bring your pages to life.
+
+### MDX plugins
+
+Internally MDX uses remark and rehype. Remark is a markdown processor powered by a plugins ecosystem. This plugin ecosystem lets you parse code, transform `HTML` elements, change syntax, extract frontmatter, and more.
```

Comment:
Should we link out to remark and rehype?

Reply:
I do link, at the bottom of the page. Id like to try and move away from millions of links within the actual content, especially external links. If they are internal links, then thats fine. But lets try to focus the readers attention. 😊